HOWTO.md

```diff
@@ -0,0 +1,79 @@
+## How to Guide
+AWS Serverless Application Model (AWS SAM) allows you to easily create and 
+manage resources used in your serverless application using AWS CloudFormation. 
+This document explains how to write SAM templates and deploy them to 
+AWS CloudFormation. 
+
+## Writing SAM Template
+Checkout the [latest specification](versions/2016-10-31.md) for details on how to write a SAM template
+
+## Packing Artifacts (Manual)
+Before you can deploy a SAM template, you should first upload your Lambda 
+function code zip and API's OpenAPI File to S3. Set the `CodeUri` and 
+`DefinitionUri` properties to the S3 URI of uploaded files. 
+Now your template is ready to be deployed.
+
+```
+MyLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+        CodeUri: s3://<mybucket>/<my-zipfile-path>
+        ...
+
+MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+        DefinitionUri: s3://<mybucket>/<my-openapi-file-path>
+        ...
+```
+
+## Packing Artifacts (Automated)
```

Comment:
I think switching the order around so automatic would make more sense. That way people who are just taking a look get the easier version first.